.gitignore

```diff
@@ -13,7 +14,16 @@ logs
 /jsettlers.common/resources/images/*
 !/jsettlers.common/resources/images/movables.txt
 /jsettlers.common/resources/save
+config.prp
 
 /jsettlers.main.swing/*.log
 
-/jsettlers.tests/out
+#IDE Files
```

Comment:
Eclipse, I suppose what I meant to say is there's no harm to add them for a few IDE's as long as they don't exclude valid files.
BTW (slight aside) I'd still be interested to assist with the UI when you decide which course of action you'd like to take.